src/GitHub.VisualStudio/TeamExplorer/Connect/GitHubConnectSection.cs

```diff
@@ -38,16 +49,32 @@ bool LoggedIn
         public bool ShowLogin
         {
             get { return showLogin; }
-            set { showLogin = value;  this.RaisePropertyChange(); }
+            set { showLogin = value; this.RaisePropertyChange(); }
         }
 
         bool showLogout;
         public bool ShowLogout
         {
             get { return showLogout; }
-            set { showLogout = value;  this.RaisePropertyChange(); }
+            set { showLogout = value; this.RaisePropertyChange(); }
+        }
+
+        ObservableCollection<ISimpleRepositoryModel> repositories;
+        public ObservableCollection<ISimpleRepositoryModel> Repositories
+        {
+            [return:AllowNull]
+            get { return repositories; }
+            set { repositories = value; this.RaisePropertyChange(); }
         }
 
+        ISimpleRepositoryModel selectedRepository;
+        [AllowNull]
+        public ISimpleRepositoryModel SelectedRepository {
+            [return: AllowNull] get { return selectedRepository; }
+            set { selectedRepository = value; this.RaisePropertyChange(); } }
```

Comment:
Nitpick Neymar: brace formatting here is inconsistent with the rest of the code. :smile: